pkg/kubelet/container/container_gc.go

```diff
@@ -39,7 +39,7 @@ type ContainerGCPolicy struct {
 // Implementation is thread-compatible.
 type ContainerGC interface {
 	// Garbage collect containers.
-	GarbageCollect() error
+	GarbageCollect(allSourcesReady bool) error
```

Comment:
@yujuhong Can we add a few comments here to explain why we add this and when are we going to remove it? Or link an issue.